robot/Cumulus/resources/NPSP.py

```diff
@@ -858,7 +858,7 @@ def wait_for_batch_to_complete(self, path, *args, **kwargs):
         locator = self.get_npsp_locator(path,*args, **kwargs)
         while True:
             i += 1
-            if i > 14:
+            if i > 20:
```

Comment:
Make sure you update the keyword documentation to match the implementation. The documentation says it checks every 15 seconds. for 3.5 minutes.  The code now waits every 20 seconds for over 6.5 minutes. It can actually wait way longer than that, since it first asks seleniun to wait, and then it adds an extra 20 seconds on each loop.